src/clarity-angular/datagrid/_datagrid.clarity.scss

```diff
@@ -542,6 +543,11 @@
         }
     }
 
+    .datagrid-foot-description {
+        display: block;
+        white-space: nowrap;
```

Comment:
If we go with `white-space: nowrap`, then we should probably handle overflow, right? Otherwise long text will be really messed up here.